ArduPlane/is_flying.cpp

```diff
@@ -289,11 +289,15 @@ void Plane::crash_detection_update(void)
         crash_state.checkedHardLanding = false;
     }
 
+#if AP_AIRSPEED_ENABLED
     // if we have no GPS lock and we don't have a functional airspeed
     // sensor then don't do crash detection
     if (gps.status() < AP_GPS::GPS_OK_FIX_3D && (!airspeed.use() || !airspeed.healthy())) {
         crashed = false;
     }
+#else
```

Comment:
this still needs the GPS check

Reply:
Removed defining out airspeed from plane in this PR. I will fix the bug in the branch I have to remove airspeed from plane.